src/actions/post-photo.ts

```
@@ -0,0 +1,53 @@
+  const token = cookies().get("token")?.value;
+
+  const name = formData.get("nome") as string | null;
+  const years = formData.get("idade") as string | null;
+  const weight = formData.get("peso") as string | null;
+  const image = formData.get("img") as File;
+
+  try {
+    if (!token) throw new Error("Usuário não autenticado.");
+    if (!name || !years || !weight || image.size === 0)
+      throw new Error("Preencha os dados.");
+    if (isNaN(Number(years)) || isNaN(Number(weight)))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)